modules/core/src/main/scala/org/scalasteward/core/buildtool/maven/command.scala

```diff
@@ -17,6 +17,6 @@
 package org.scalasteward.core.buildtool.maven
 
 object command {
-  val listDependencies = "dependency:list"
+  val listDependencies = "dependency:list -DexcludeTransitive=true"
```

Comment:
This needs to be a separate argument in https://github.com/scala-steward-org/scala-steward/blob/d3e22af810fa71b8917fd47e5ba128b461b8b197/modules/core/src/main/scala/org/scalasteward/core/buildtool/maven/MavenAlg.scala#L45
Maven will fail otherwise with `[info] [ERROR] Could not find goal 'list -DexcludeTransitive=true'`.

Reply:
Ah, how do the `MavenAlgTest` pass then? I guess I would have expected that to blow up in this case.

Reply:
`MavenAlgTest` unfortunately does not actually run Maven but only checks that it would call `mvn` with the expected args.